edgepi-dac.html

```diff
@@ -18,48 +18,44 @@
       return this.name || "dac";
     },
     oneditprepare: function () {
-      // Query html elements
       const configTransportSelect = document.getElementById(
         "node-input-transport"
       );
-      const gainOption1 = document.getElementById("node-input-gain1");
-      const gainOption2 = document.getElementById("node-input-gain2");
+      const gain1 = document.getElementById("node-input-gain1");
+      const gain2 = document.getElementById("node-input-gain2");
```

Comment:
what is gain1 and gain2? I am confused : /

Reply:
Updated to gainOn and gainOff